docs/src/pages/components/buttons/UnstyledButtons.js

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import Stack from '@material-ui/core/Stack';
+import ButtonUnstyled, {
+  buttonUnstyledClasses,
+} from '@material-ui/unstyled/ButtonUnstyled';
+import { styled } from '@material-ui/system';
+
+const StyledButtonRoot = styled('span')(`
```

Comment:
Is it intentional that we have `span` here? Isn't `button` more appropriate?

Reply:
This is to show that you can provide another element and the ButtonUnstyled will take care of handling tab index, keyboard support and aria attributes. I'll add a simpler demo as well to showcase just the standard button.